modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown for a node, create a `KubeletConfig` custom resource (CR) to specify a grace period and shutdown profile for specific nodes. If graceful node shutdown is not configured, the default grace period is `None` and the pod is restarted without running PreStop.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Define a `KubeletConfig` CR with graceful node shutdown:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: graceful-shutdown
+  namespace: openshift-machine-config-operator
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+  kubeletConfig:
+    shutdownGracePeriod: "3m" <2>
+    shutdownGracePeriodCriticalPods: "2m"
+EOF
+----
+
+<1> Apply graceful node shutdown to all worker nodes.
+<2> Define a shutdown period of 3 minutes for regular pods.
+<3> Define a grace period for critical pods of 2 minutes.
+
+
+. Create the `KubeletConfig` object:
```

Comment:
this line should not be indented to the right, and it is marked no.1 (which is duplicate with previous "Define a `KubeletConfig` CR with graceful node shutdown" )

Reply:
Fixed.